testing/meek/APKBUILD

```diff
@@ -0,0 +1,52 @@
+# Contributor: kpcyrd <git@rxv.cc>
+# Maintainer: kpcyrd <git@rxv.cc>
+pkgname=meek
+pkgver=0.30
+pkgrel=0
+pkgdesc="A pluggable transport proxy written in Go"
+url="https://trac.torproject.org/projects/tor/wiki/doc/meek"
+arch="all"
+license="CC0 1.0 Universal"
+depends="ca-certificates"
+makedepends="go"
+subpackages="$pkgname-doc"
+# no test suite available
+options="!check"
+source="$pkgname-$pkgver.tar.gz::https://gitweb.torproject.org/pluggable-transports/$pkgname.git/snapshot/$pkgname-$pkgver.tar.gz"
+builddir="$srcdir/$pkgname"
+
+prepare() {
+	mkdir -p ${builddir%/*}
+	mv "$srcdir/$pkgname-$pkgver" "$builddir/"
+	cd "$builddir"
+
+	export GOPATH="$srcdir"
+	(cd meek-client && go get -v -d)
+	(cd meek-server && go get -v -d)
+
+	default_prepare
+}
+
+build() {
+	cd "$builddir"
+	export GOPATH="$srcdir"
+	go build -v -o meek-client/meek-client ./meek-client/...
+	go build -v -o meek-server/meek-server ./meek-server/...
+}
+
+package() {
+	cd "$builddir"
+	install -Dm 755 meek-client/meek-client "$pkgdir/usr/bin/meek-client"
+	install -Dm 755 meek-server/meek-server "$pkgdir/usr/bin/meek-server"
+	setcap 'cap_net_bind_service=+ep' "$pkgdir/usr/bin/meek-server"
```

Comment:
Don't really know if this is results in a change in the final package. You may have to set this in a post-install/post-upgrade function.

Reply:
Those are preserved by the package. There are a few packages in aports that use setcap, all of them set it in APKBUILD instead of a post-install/post-upgrade.

Reply:
You need to depend on libcap to be able to do this.

Reply:
Done